cachi2/core/package_managers/rpm/main.py

```diff
@@ -378,3 +380,37 @@ def _generate_repofiles(
 
             with open(repo_file_path, "w") as f:
                 repofile.write(f)
+
+def _get_ssl_context():
+    client_cert = getenv("C2_CLIENT_CERT")
+    client_key = getenv("C2_CLIENT_KEY")
+    ca_bundle = getenv("C2_CA_BUNDLE")
```

Comment:
I'd say the variable names are 'made up with inspiration from the SSL module' (for example it uses the CA 'bundle' terminology'. I support moving the env variables to CLI, I just hadn't figured out how yet but I have now.

For encrypted private keys are you expecting a user to enter that password interactively?

Reply:
First of all, sorry for a late reply.

>  I support moving the env variables to CLI, I just hadn't figured out how yet but I have now.

This would ideally be done via extra options in the input JSON: https://github.com/containerbuildsystem/cachi2/blob/1be1ab417de0a1bc43db386889e5023ebd05983f/cachi2/core/models/input.py#L173 because this way it's only ever viable for a given package manager, not globally like it is for Golang. It'll need to be extended appropriately.
Even though it's generic enough for other package manager's use, that feature hasn't been enabled globally, so I'd be willing to accept tweaks only to the RPM related model and expand it to other ones in the future if needed.

> For encrypted private keys are you expecting a user to enter that password interactively?

No, the standard way to go about that is password files.

Reply:
the cli options I've added are pretty standard / generic to SSL and cllient TLS. I think anyone with some experience will recognize them.